client/influxdb.go

```diff
@@ -124,11 +124,15 @@ func (c *Client) Write(bp BatchPoints) (*Response, error) {
 
 	var b bytes.Buffer
 	for _, p := range bp.Points {
-		if p.Raw != "" {
+		if p.Raw != "" && len(bp.Tags) == 0 {
```

Comment:
Ah, no, it wasn't. I figured `Raw` was there if the Point had previously been marshaled here and the other data was also available. If it's the case that `Raw` might be the only thing set, we could either ignore `BatchPoint.Tags` or parse and re-emit.